specs/Appearance.md

```diff
@@ -0,0 +1,114 @@
+
+Please note this API will only set the overall appearance, but not theme as defined in the Edge browser.
+For reference, in the screenshot below, this API is meant to expose the Overall Appearance section as a WebView2 API.
+
+![Alt text](./media/EdgeSettingsAppearance.png "a title")
+# Examples
+
+## C++
+
+```cpp
+wil::com_ptr<ICoreWebView2Profile> m_profile;
+
+void ViewComponent::SetTheme(COREWEBVIEW2_THEME_KIND value)
+{
+    wil::com_ptr<ICoreWebView2Profile> webViewProfile;
+
+    CHECK_FAILURE(m_profile->QueryInterface(IID_PPV_ARGS(&webViewProfile)));
+    if (webViewProfile)
+    {
+        CHECK_FAILURE(webViewProfile->put_Theme(value));
+    }
+}
+```
+
+## C#
+
+```c#
+
+private CoreWebView2Profile m_profile;
+
+void SetTheme(CoreWebView2ThemeKind value)
+{
+    // Check for runtime support
+    try
+    {
+        m_profile.Theme = value;
+    }
+    catch (NotImplementedException exception)
+    {
+        MessageBox.Show(this, "Set Theme Failed: " + exception.Message,
+                        "Set Theme");
+    }
+}
+
+```
+
+# API Details
+## C++
+```cpp
+[uuid(5f817cce-5d36-4cd0-a1d5-aaaf95c8685f), object, pointer_default(unique)]
+interface ICoreWebView2Controller4 : ICoreWebView2Controller3 {
+  /// The Theme property sets the overall theme of the WebView2 instance.
+  /// The input parameter is either COREWEBVIEW2_THEME_KIND_SYSTEM,
+  /// COREWEBVIEW2_THEME_KIND_LIGHT, or COREWEBVIEW2_THEME_KIND_DARK.
+  /// Note that this API applies theme to WebView2 pages, dialogs, menus,
+  /// and sets the media feature `prefers-color-scheme` for websites to respond to.
+  ///
+  /// \snippet ViewComponents.cpp SetAppearance
+  /// Returns the value of the `Appearance` property.
+  [propget] HRESULT Theme(
+    [out, retval] COREWEBVIEW2_THEME_KIND* value);
+
+  /// Sets the `Theme` property.
+  [propput] HRESULT Theme(
+    [in] COREWEBVIEW2_THEME_KIND value);
+}
+
+/// An enum to represent the options for WebView2 Theme: system, light, or dark.
+[v1_enum]
+typedef enum COREWEBVIEW2_THEME_KIND {
+  /// System theme
+  COREWEBVIEW2_THEME_KIND_SYSTEM,
+
+  /// Light theme
+  COREWEBVIEW2_THEME_KIND_LIGHT,
+
+  /// Dark theme
+  COREWEBVIEW2_THEME_KIND_DARK
+} COREWEBVIEW2_THEME_KIND;
+```
+
+### C#
+```c#
+namespace Microsoft.Web.WebView2.Core
+{
+    [doc_string("An enum to represent the options for WebView2 Theme: system, light, or dark.")]
+    enum CoreWebView2ThemeKind
+    {
+        [doc_string("System theme.")]
+        System = 0,
+        [doc_string("Light theme.")]
+        Light = 1,
+        [doc_string("Dark theme.")]
+        Dark = 2,
+    };
+
+    runtimeclass CoreWebView2Profile
+    {
+        // ...
+
+        [interface_name("Microsoft.Web.WebView2.Core.CoreWebView2Profile")]
```

Comment:
IIRC the names here match the COM interface name right?

Reply:
yes they do